content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/sboms-and-attestations.md

```diff
@@ -0,0 +1,89 @@
+| Policies can be applied to SBOM | no | yes |
+| Uploads OCI artifact to registry | yes | yes |
+| OCI artifact extension | `.sbom` | `.att` |
+
+The remaining sections of this guide elaborate further on the similarities and differences between these two commands.
+
+
+## SBOMs vs. Attestations
+
+An SBOM is essentially an electronic packing slip: it's a list of all the components that went into making a given piece of software. But unless you have some indication of when the software was produced, who produced it, and how it was produced, then you can't say with any certainty that the components listed in the SBOM are actually part of the software you're running.
+
+An *attestation* allows the end users or consumers of a software artifact (in the context of this guide, an SBOM) to verify the quality of that artifact independently from the producer of the software. It also requires software producers to provide verifiable proof of the quality of their software.
+
+Put differently, an attestation is a written assurance of a software artifact's *provenance*, or the verifiable information about the artifact describing where, when, and how it was produced. You can think of an attestation as a proclamation that "software artifact X" was produced by "person Y" at "time Z".
+
+Both `cosign attest` and `cosign attach` associate an artifact with an image and upload it to a registry. However, `cosign attest` generates an [in-toto attestation](https://in-toto.io/) while `cosign attach` does not. `cosign attest` then attaches it to the provided image and uploads it to a registry as an OCI artifact with a `.att` extension.
+
+In the following example, `image.sbom` is an SBOM file that was previously created, `$IMAGE` is the image that will be attached to the SBOM, and `cosign.key` is the signer's private key.
+
+```sh
+cosign attest --key cosign.key --predicate image.sbom $IMAGE
+```
+
+Note that after creating an attestation, you can verify it with Cosign's `verify-attestation` subcommand.
+
+```sh
+cosign verify-attestation $IMAGE
+```
+
+`cosign attach`, on the other hand, only attaches an SBOM to an image and uploads it to a registry. In this case the SBOM isn't signed, meaning that Chainguard Enforce won't have any way to verify its authenticity.
+
+```sh
+cosign attach sbom --sbom image.sbom $IMAGE
+```
+
+This will upload the SBOM to the registry as an OCI artifact with a `.sbom` extension.
+
+Be aware that there is also the `cosign sign` command. After running `cosign attach` to attach an SBOM and upload it to a registry, you can then run `cosign sign` to sign the SBOM, and upload the signature to the registry as a separate OCI artifact, this time with the `.sig` extension.
+
+
+## Should I attest or attach?
+
+Attestations are generally seen as being more trustworthy than unsigned SBOMs, because you can identify who signed them and when. However, Chainguard Enforce will not ingest an attestation unless there is an active policy covering it. 
```

Comment:
again, this may have changed. 

either way, perhaps me mention what "ingest" means, here or maybe above in the table section